src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -209,7 +281,7 @@ public function bulkinsert(Table $table, $rows)
         }
 
         $count_keys = count($keys);
-        $query = "(" . implode(', ', array_fill(0, $count_keys, '?')) . ")";
+        $query = '(' . implode(', ', array_fill(0, $count_keys, '?')) . ')';
```

Comment:
I think the same thing can be said here. I think we could do like the following:
```php
if ($this->isDryRunEnabled()) {
    $values = array_map(function ($row) {
        return '(' . implode(', ', array_map([$this, 'quoteValue'], $row)) . ')';
    }, $rows);
    
    ...
} else {
    $values = [];
    foreach ($rows as $row) {
        foreach ($row as $v) {
            $values[] = $v;
        }
    }

    ...
}
```